docs/docs-content/clusters/edge/site-deployment/kubevip.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_label: "Publish Cluster Services with Kube-vip"
+title: "Publish Cluster Services with Kube-vip"
+description: "Guide to publishing cluster services with kube-vip."
+hide_table_of_contents: false
+sidebar_position: 30
+tags: ["edge"]
+---
+
+You can use kube-vip to provide a virtual IP address for your cluster and use it to expose a service of type `LoadBalancer` on the external network.
+
+Kube-vip supports DHCP environments and can request additional IP address from the DHCP server automatically. Using kube-vip, you can expose services inside your cluster externally with a virtual IP address even if you do not have control over your host's network. Kube-vip can also act as a load balancer for both your control plane and Kubernetes services of type `LoadBalancer`.
+
+## Prerequisites
+
+- At least one Edge device with x86_64/AMD64 processor architecture registered in your Palette account 
+
+## Enablement
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu** click **Clusters > Add a New Cluster**.
+
+4. Choose **Edge Native** for the cluster type and click **Start Edge Native Configuration**.
+
+5. Give the cluster a name, description, and tags. Click on **Next**.
+
+6. Select the cluster profile that you plan to use to deploy your cluster and publish services with and click **Next**.  
```

Comment:
```suggestion
6. Select the cluster profile you plan to use to deploy your cluster, publish services with, and click **Next**.  
```
Lenny, this sentence reads a bit awkward. Take a peek at the suggestion and see what you think.